config/install/field.field.paragraph.oe_facts_figures.field_oe_paragraphs.yml

```diff
@@ -6,42 +6,83 @@ dependencies:
     - paragraphs.paragraphs_type.oe_fact
     - paragraphs.paragraphs_type.oe_facts_figures
   module:
+    - composite_reference
     - entity_reference_revisions
+third_party_settings:
```

Comment:
where is this coming from ?

Reply:
It is the first commit, "Re-save the field".
Whenever we do this, we get noise based on the current state of other components active in the package.
You can compare different field.field.* config files, they have different noise depending when they were saved.
- https://github.com/openeuropa/oe_paragraphs/blob/dd7ab432f3431742016fbb2e560b6a0dd86eaf44/config/install/field.field.paragraph.oe_list_item_block.field_oe_paragraphs.yml
- https://github.com/openeuropa/oe_paragraphs/blob/dd7ab432f3431742016fbb2e560b6a0dd86eaf44/modules/oe_paragraphs_illustrations_lists/config/install/field.field.paragraph.oe_illustration_list_icons.field_oe_paragraphs.yml

Reply:
I removed the noise, following our convo.

Reply:
so usually on a website project we would want the noise, to "normalize" the config.
But you are right in this case it is pointless to attempt to normalize because other projects that use oe_paragraphs won't have the same fields and modules.